tests/php/Model/SiteTreeTest_CMSActions.php

```diff
@@ -0,0 +1,241 @@
+<?php
+
+namespace SilverStripe\CMS\Tests\Model;
+
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\Subsites\Extensions\SiteTreeSubsites;
+use TractorCow\Fluent\Extension\FluentSiteTreeExtension;
+
+class SiteTreeTest_CMSActions extends SapphireTest
+{
+    protected static $fixture_file = 'SiteTreeTest.yml';
+
+    protected static $illegal_extensions = [
+        SiteTree::class => [
+            SiteTreeSubsites::class,
+            FluentSiteTreeExtension::class,
+        ],
+    ];
+
+    protected static $extra_dataobjects = [
+        SiteTreeTest_ClassA::class,
+        SiteTreeTest_ClassB::class,
+        SiteTreeTest_ClassC::class,
+        SiteTreeTest_ClassD::class,
+        SiteTreeTest_ClassCext::class,
+        SiteTreeTest_NotRoot::class,
+        SiteTreeTest_StageStatusInherit::class,
+        SiteTreeTest_DataObject::class,
+    ];
+
+    public function testGetCMSActions()
```

Comment:
Could you please move `testGetCMSActions()` and `testGetCMSActionsWithoutForms()` to SiteTreeTest.php and remove SiteTreeTest_CMSActions.php.  

I know that SiteTreeTest.php is already a giant file, though adding these new tests to the existing file is consistent with our naming convention of <classname>Test.php

Reply:
Done.